Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.h

```diff
@@ -70,7 +73,8 @@ typedef void(^VOKManagedObjectsReturnBlock)(NSArray *arrayOfManagedObjects);
  @param contextOfNil    The managed object context to update and/or insert the object. If nil, the main context will be used.
  @return                An instance of this subclass of NSManagedObject.
  **/
-+ (instancetype)vok_addWithDictionary:(NSDictionary *)inputDict forManagedObjectContext:(NSManagedObjectContext *)contextOrNil;
++ (nullable instancetype)vok_addWithDictionary:(nullable NSDictionary *)inputDict
```

Comment:
Should the dictionary actually be nullable? If you're trying to import a nonexistent dictionary, that's just gonna fail anyway.